models/snake_and_ladder.py

```diff
@@ -0,0 +1,86 @@
+__author__ = "Shakti Prasad Lenka"
+__email__ = "sinulenka001@gmail.com"
+
+from random import randrange
+from .player import Player
+
+class SnakeAndLadder(object):
+
+	def __init__(self,boardSize,nDice,snakes,ladder,players = []):
+		self._boardSize = boardSize
+		self._nDice = nDice
+		self._players = players
+		self._snakes = snakes
+		self._ladder = ladder
+		self._nPlayers = len(players)
+		self._nRounds = 0
+
+	def diceThrow(self):
+		value = 0
+		for throw in range(self._nDice):
+			value += randrange(1,6)
+		return value
+
+	def ouput(self,player,diceValue):
+		print("{} rolled a {} and moved from {} to {}".format(
+														player.name,
+															diceValue,
+												player.initialPosition,
+												player.finalPosition))
+
+	def oneRound(self):
+		winnerIndex = -1
+		winner =None
+		success = False
+		for index,player in enumerate(self._players):
+			damnSnake = True
+			yayLadder = True
+
+			#Lets throw dice on behalf of player.name
+			diceValue = self.diceThrow()
+
+			player.initialPosition = player.finalPosition
+			#only change position if final position is less than _boardSize
+			if player.finalPosition + diceValue <=self._boardSize:
+				player.finalPosition += diceValue
+
+
+			##check for ladder and snakes
+
+			while damnSnake or yayLadder:
+				if player.finalPosition in self._snakes:
+					player.finalPosition = self._snakes[player.finalPosition]
+				elif player.finalPosition in self._ladder:
+					player.finalPosition = self._ladder[player.finalPosition]
+
+				#check for snake or ladder in resulting position
+				damnSnake = player.finalPosition in self._snakes
+				yayLadder = player.finalPosition in self._ladder
+
+			#print results
+			self.ouput(player, diceValue)
+
+			#check success
+			#let other players complete the round even after one player
```

Comment:
What if two players won in the same round. My code will display only the first winner in this round and second winner in next round. Need to fix this.